extras/multisigs.json

```diff
@@ -85,6 +86,18 @@
     "lm": "0x040E995520F92F96142d1a76c16D4af21A2eFDE7",
     "maxi_ops": "0x040E995520F92F96142d1a76c16D4af21A2eFDE7"
   },
+  "mode": {
+    "lm": "0x9ff471F9f98F42E5151C7855fD1b5aa906b1AF7e",
+    "dao": "0x4f22C2784Cbd2B24a172566491Ee73fee1A63c2e"
```

Comment:
idem for linea and fraxtal

Reply:
I think a lot of scripts use lm/dao.  Those are functional names.  In the case of gnosis chain, the safe is not functional.  At some point it may become the dao multisig, at which case I would suggest the current one became old_dao